bigframes/session/executor.py

```diff
@@ -108,6 +110,7 @@ def __init__(
         ] = weakref.WeakKeyDictionary()
         self.metrics = metrics
         self.bqstoragereadclient = bqstoragereadclient
+        self.nondeterminism_only_once = (nondeterminism_only_once,)
```

Comment:
self.nondeterminism_only_once seems not used?

Reply:
sorry, leaked from some other changes I was working on. removed from new revision

Reply:
I don't see new commit, maybe you didn't push?

Reply:
didn't actually push, now its pushed